Tests/scripts/hook_validations/integration.py

```diff
@@ -488,3 +489,14 @@ def is_docker_image_changed(self):
                 return True
 
         return False
+
+    def is_valid_id(self):
+        """Validate that the id equals to the name."""
+        _id = self.current_integration.get('commonfields', {}).get('id')
```

Comment:
should be `id_`

https://www.python.org/dev/peps/pep-0008/#function-and-method-arguments